wagtail/admin/templates/wagtailadmin/panels/inline_panel_child.html

```diff
@@ -2,17 +2,25 @@
 
 {% fragment as id %}inline_child_{{ child.form.prefix }}{% endfragment %}
 {% fragment as panel_id %}{{ id }}-panel{% endfragment %}
-<div data-inline-panel-child id="{{ id }}" data-contentpath-disabled>
+<div data-w-formset-target="child" id="{{ id }}" data-contentpath-disabled data-w-orderable-target="item" data-w-orderable-item-id="{{ id }}">
```

Comment:
Please move the `id` to before the data attributes, as per our style guide.

https://docs.wagtail.org/en/stable/contributing/ui_guidelines.html#html-guidelines